sormas-e2e-tests/src/test/java/org/sormas/e2etests/entities/services/ImmunizationService.java

```diff
@@ -68,4 +68,14 @@ public Immunization buildGeneratedImmunizationWithMeansOfImmunizationFromCase(
         .meansOfImmunization(meansOfImmunization)
         .build();
   }
+
+  public Immunization buildImmunizationWithSpecificResponsibleLocation() {
```

Comment:
let's parametrize this method to allow reuse it also for other combinations of region+district

Reply:
ok, I'll parametrize it

Reply:
done